src/engine/test/integration_tests/configuration_files/config.env

```diff
@@ -38,3 +38,8 @@ WAZUH_INDEXER_DB_PATH=ENV_PATH_PLACEHOLDER/indexer-connector
 # Configure archiver
 WAZUH_ARCHIVER_PATH=ENV_PATH_PLACEHOLDER/archives.json
 WAZUH_ARCHIVER_ENABLED=false
+
+# Configure User and Group
+WAZUH_ENGINE_USER=runner
+WAZUH_ENGINE_GROUP=docker
```

Comment:
Use a placeholder for this, should be automatically detected and use de current user and group.